sirepo/package_data/static/html/radia-source.html

```diff
@@ -8,7 +8,7 @@
       </div>
       </div>
   <div class="col-md-12" data-ng-if="src.showParams()">
-    <!--<div data-advanced-editor-pane="" data-view-name="src.getMagnetType()" data-parent-controller="src"></div>-->
-    <div data-basic-editor-panel="" data-view-name="hybridUndulator" data-parent-controller="src" data-panel-title="Undulator"></div>
+    <div data-ng-if="src.getMagnetType() == 'undulator'" data-basic-editor-panel="" data-view-name="hybridUndulator" data-parent-controller="src" data-panel-title="Undulator"></div>
+    <!--<div data-ng-if="src.getMagnetType() == 'dipole'" data-basic-editor-panel="" data-view-name="dipole" data-parent-controller="src"></div>-->
```

Comment:
Can the code be removed since it is commented out?

Reply:
738b063

Agreed, I'll put it back when needed